src/SettingsController.js

```diff
@@ -59,6 +59,8 @@ class SettingsController extends AbstractController {
   refresh () {
     const { schema, mapping, computed = {} } = this
 
+    if(!fs.existsSync(path.join(process.cwd(), '.env'))) fs.writeFileSync(path.join(process.cwd(), '.env'), "", { flag: 'wx' })
```

Comment:
A note on style: we never use inline if statements, it affects legibility. Rather expand it:
```javascript
if (...) {
  // ...
}
```

Reply:
I'll update this.